data/datasets/credit_card_default_loader.py

```diff
@@ -0,0 +1,88 @@
+from typing import Optional
+import pandas as pd
+from data.datasets import base_loader
+from data.datasets import utils
+
+
+DATASET_NAME = "credit_card_default"
+_SOURCE_TARGET_COLUMN = "default payment next month"
+_URL = (
+    "https://archive.ics.uci.edu/ml/machine-learning-databases/00350/"
+    "default%20of%20credit%20card%20clients.xls"
+)
+
+
+class CreditCardDefaultLoader(base_loader.DataLoader):
+    """Loads the UCI Credit Card Default dataset from the internet or local
```

Comment:
It's useful to include a link to the dataset description here. Unless there are dataset columns you need to handle in a particular way, it may/may not be necessary to include a dataset description as you have it below. It would be useful to describe the resulting structure of the dataset post-loading, especially if there are any column transformations.

Reply:
https://archive.ics.uci.edu/ml/datasets/default+of+credit+card+clients